src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -696,14 +694,14 @@ private void spawnCitizen(CitizenData data)
     }
     
     /**
-     * Checks if the achievements are valid
+     * Checks if the achievements are valid.
```

Comment:
Valid in what form?

What are valid achievements, what are invalid? What does it do if they are invalid as this method does return nothing. Make it so I don't have to read the code to understand this.